corehq/apps/accounting/templates/accounting/partials/confirm_plan_summary.html

```diff
@@ -57,7 +57,16 @@ <h4>
     </h4>
   {% endif %}
 
-  {% if is_upgrade %}
+  {% if is_annual_plan %}
+    <p class="text-free">
```

Comment:
worth bringing `p` tag out of conditional

Reply:
I didn't move the <p> tags out of the conditional here because there are a couple that have multiple paragraph blocks, specifically here https://github.com/dimagi/commcare-hq/blob/e60d3cf73a31e849e8434e1b790e3ae6846bf3a6/corehq/apps/accounting/templates/accounting/partials/confirm_plan_summary.html#L78-L79